test/parallel/test-spawn-cmd-named-pipe.js

```diff
@@ -42,10 +42,8 @@ if (!process.argv[2]) {
     assert.fail('Failed to get COMSPEC');
   }
 
-  const args = ['/c', process.execPath, __filename, 'child',
-                '<', stdinPipeName, '>', stdoutPipeName];
-
-  const child = spawn(comspec, args);
+  const child = exec(`"${process.execPath}" "${__filename}" child < ${
```

Comment:
The variant in https://github.com/nodejs/node/pull/12945#discussion_r115816593 seems reasonable enough.